homeassistant/components/velbus/services.py

```diff
@@ -46,20 +46,20 @@ async def scan(call: ServiceCall) -> None:
         """Handle a scan service call."""
         entry = get_config_entry(call.data[CONF_INTERFACE])
         if entry:
-            await entry.runtime_data.cntrl.scan()
+            await entry.runtime_data.controller.scan()
```

Comment:
Not directly linked to this PR, but is there a reason why you are not raising a ServiceValidationError if the entry is not found?

Reply:
This is on the to-do list, will do it in a follow-up PR